CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v4.6.2 (2023-05-15)
+
+- Rollback RestSharp dependency range extension due to breaking changes with RestSharp between v108 and v110. This library only works with RestSharp v108.
```

Comment:
issue: please move the items from 4.6.1 to here and leave a note under 4.6.1 stating it shouldn't be used